external/common/include/fast_cpp_csv_parser/LICENSE

```diff
@@ -0,0 +1,28 @@
+Copyright (c) 2015, ben-strasser
```

Comment:
where should the LICENSE file be placed? Is it OK to keep within the `external/common/include/"..."` dir?